src/pylibsshext/session.pyx

```diff
@@ -146,6 +149,7 @@ cdef class Session(object):
     def set_ssh_options(self, key, value):
         cdef unsigned int port_i
```

Comment:
Can we use generic variable names here. Since `set_ssh_options` is called for each option in loop the cdef variables are used to store the value to apporicate datatype. How about `value_int`, `value_long_int` `value_unsigned_int`?

Reply:
Yeah, that was what I was thinking would be a good path for these variables.